code/modules/clothing/suits/armor.dm

```diff
@@ -79,7 +79,7 @@
 	desc = "An armored vest of dubious quality. Provides decent protection against physical damage, for a piece of crap."
 	icon_state = "armor_handmade"
 	armor_list = list(
-		melee = 30,
+		melee =7,
 		bullet = 20,
 		energy = 15,
```

Comment:
Bullet and Energy armor need to be adjusted down to make this not god armor

Reply:
ah piss